File: Python/mini-max-sum.py
import math

def miniMaxSum(arr):
    mini = math.inf
    maxi = 0
    sum_val = 0
    
    for num in arr:
        sum_val += num
        mini = min(mini,num)
        maxi = max(maxi,num)
        
    print(sum_val-maxi,sum_val-mini)

File: Java/mini-max-sum.java
class Result {
    public static void miniMaxSum(List<Integer> arr) {
        long total = 0, min = Long.MAX_VALUE, max = 0;
        for(int num : arr) {
            total += num;
            min = Math.min(min, num);
            max = Math.max(max, num);
        }
        System.out.println((total-max) + " " + (total-min));
    }
}